src/models/UserProfileModel.ts

```diff
@@ -59,6 +59,11 @@ export class UserProfileModel extends Model {
         FETCH_CLICKED_DOCUMENT_FAIL: 'Fetching clicked documents details failed'
     });
 
+    private static readonly MODEL_CONFIG = {
+        customAttribute: true,
+        silent: true
```

Comment:
Are you sure it needs to be silent ? Won't that prevent sending updates or something ?

Reply:
That's what we want